packages/mobile/src/home/NotificationBox.tsx

```diff
@@ -199,6 +213,45 @@ export class NotificationBox extends React.Component<Props, State> {
       })
     }
 
+    for (const [id, notification] of Object.entries(this.props.extraNotifications)) {
+      if (!notification || notification.dismissed) {
+        continue
+      }
+      const texts = notification[
+        i18n.language.slice(0, 2) as keyof Notification
```

Comment:
I'm thinking we should use the full language tag instead so it matches what we do elsewhere for localized data.

Reply:
Hmm, on Firebase we have the keys `en`, `es` and `pt`, are you suggesting having different translations for every country? Like separating `en-US` from `en-GB`?

Reply:
probably best to have both the more specific and generic ways work. check lang-cc first then fall back to just lang.

Reply:
I don't think it's likely that we'll use that, but I don't see why not have it anyways, I added the full language as default and the 2 letter code as fallback :)